templates/patient/card/manage_care_team.html.twig

```diff
@@ -0,0 +1,111 @@
+                    <td><select name="team[${idx}][status]" class="form-control form-control-sm">${statusOptions}</select></td>
+                    <td><input name="team[${idx}][note]" value="${member.note || ''}" class="form-control form-control-sm"></td>
+                    <td><button type="button" class="btn btn-danger btn-sm" onclick="this.closest('tr').remove()">×</button></td>
+                `;
+                document.querySelector('#care_team_table tbody').appendChild(row);
+                row.querySelector(`select[name="team[${idx}][user_id]"]`).value = member.user_id;
+                row.querySelector(`select[name="team[${idx}][facility_id]"]`).value = member.facility_id;
+                row.querySelector(`select[name="team[${idx}][role]"]`).value = member.role;
+                row.querySelector(`select[name="team[${idx}][status]"]`).value = member.status;
+            });
+
+            if (existingCareTeam[0]?.team_name) {
+                document.getElementById("team_name").value = existingCareTeam[0].team_name;
+            }
+        }
+
+        function addTeamRow() {
+            const idx = document.querySelectorAll('#care_team_table tbody tr').length;
+            const row = document.createElement('tr');
+            row.innerHTML = `
+                <td><select name="team[${idx}][user_id]" class="form-control form-control-sm">${userOptions}</select></td>
+                <td><select name="team[${idx}][role]" class="form-control form-control-sm">${roleOptions}</select></td>
+                <td><select name="team[${idx}][facility_id]" class="form-control form-control-sm">${facilityOptions}</select></td>
+                <td><input type="text" name="team[${idx}][provider_since]" class="datepicker-new form-control form-control-sm"></td>
+                <td><select name="team[${idx}][status]" class="form-control form-control-sm">${statusOptions}</select></td>
+                <td><input name="team[${idx}][note]" class="form-control form-control-sm"></td>
+                <td><button type="button" class="btn btn-danger btn-sm" onclick="this.closest('tr').remove()">×</button></td>
+            `;
+            document.querySelector('#care_team_table tbody').appendChild(row);
+            $(row).find('.datepicker-new').datetimepicker({
+                timepicker: false,
+                formatDate: 'Y-m-d',
+                format: 'Y-m-d'
+            }).removeClass('datepicker-new').addClass('datepicker');
+        }
+
+        document.addEventListener("DOMContentLoaded", preloadTeamRows);
+    </script>
+    </head>
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-12">
+                <form method="POST" action="" onsubmit="return confirm('{{ translations.save_care_team_confirm }}');" class="mt-3">
+                    <input type="hidden" name="pid" value="{{ pid }}">
```

Comment:
need to escape above and bunch of stuff below (also would escape the translations stuff here instead of in the array creation above)